java/openjdk11/Portfile

```diff
@@ -5,11 +5,12 @@ PortSystem          1.0
 name                openjdk11
 # https://github.com/openjdk/jdk11u/tags
 version             11.0.14.1
-revision            1
+revision            2
 categories          java devel
 platforms           darwin
 supported_archs     x86_64
 license             GPL-2+
+license_noconflict  autoconf gmake
```

Comment:
These shouldn't be needed for any of your openjdkXXX ports. Indeed, they should never cause a license conflict.

You can verify the license status for any of your ports, via utility 'port_binary_distributable.tcl'. It's available via repo `macports-infrastructure`:

```
$ jobs/port_binary_distributable.tcl -v openjdk11
"openjdk11" is distributable
$ jobs/port_binary_distributable.tcl -v openjdk13
"openjdk13" is distributable
```

Reply:
Done